<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JointSideWidget</class>
 <widget class="QWidget" name="JointSideWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>213</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add new joint</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item row="0" column="0">
    <widget class="GroupBoxExtension" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="title">
      <string>Joint</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="TreeWidgetExtension" name="treeViewJoints">
        <property name="indentation">
         <number>10</number>
        </property>
        <property name="rootIsDecorated">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>30</number>
        </property>
        <item>
         <widget class="BaseEntityNameEdit" name="addName" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupBoxExtension</class>
   <extends>QGroupBox</extends>
   <header>GUI/WidgetExtensions/GroupBoxExtension.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BaseEntityNameEdit</class>
   <extends>QWidget</extends>
   <header location="global">GUI/BasicWidgets/BaseEntityNameEdit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TreeWidgetExtension</class>
   <extends>QTreeWidget</extends>
   <header>GUI/WidgetExtensions/TreeWidgetExtension.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
